packages/main/src/plugin/experimental-feature-feedback-handler.spec.ts

```
@@ -0,0 +1,216 @@
+
+const configuration: Configuration = {
+  get: configurationGetMock,
+  has: () => true,
+  update: () => Promise.resolve(),
+};
+
+class TestExperimentalFeatureFeedbackHandler extends ExperimentalFeatureFeedbackHandler {
+  override experimentalFeatures: Map<string, ExperimentalConfiguration> = new Map<string, ExperimentalConfiguration>();
+
+  override setTimestamp(feature: string, days: Timestamp): void {
+    return super.setTimestamp(feature, days);
```

Biome (2.1.2)

[error] The function should not return a value because its return type is void.

The function is here:

'void' signals the absence of value. The returned value is likely to be ignored by the caller.

(lint/correctness/noVoidTypeReturn)